enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/BackButton/BackButton.tsx

```diff
@@ -22,9 +23,9 @@ export type InteractiveQuestionBackButtonProps = Omit<
  * @category InteractiveQuestion
  * @param props
  */
-export const BackButton = ({
-  ...actionIconProps
-}: InteractiveQuestionBackButtonProps) => {
+export const BackButton: SdkFunctionalComponent<
+  InteractiveQuestionBackButtonProps
+> = ({ ...actionIconProps }: InteractiveQuestionBackButtonProps) => {
```

Comment:
Should/could we remove type from the implementation? Now we specify props via the explicit type of the function already.

If we do that we only need to change type in a single place if we want to.

```suggestion
> = ({ ...actionIconProps }) => {
```

Also, not sure why we're spreading this...

```suggestion
> = (actionIconProps) => {
```

This comment applies to all components in this PR.